input/postgres/relation_stats.go

```diff
@@ -156,3 +170,51 @@ func GetIndexStats(db *sql.DB, postgresVersion state.PostgresVersion, ignoreRege
 
 	return
 }
+
+func GetColumnStats(logger *util.Logger, db *sql.DB, globalCollectionOpts state.CollectionOpts, systemType string) (columnStats state.PostgresColumnStatsMap, err error) {
+	var sourceTable string
+
+	helperExists := false
+	db.QueryRow(QueryMarkerSQL + columnStatsHelperSQL).Scan(&helperExists)
+
+	if helperExists {
+		logger.PrintVerbose("Found pganalyze.get_column_stats() stats helper")
+		sourceTable = "pganalyze.get_column_stats()"
+	} else {
+		if systemType != "heroku" && !connectedAsSuperUser(db, systemType) && globalCollectionOpts.TestRun {
+			logger.PrintInfo("Warning: Limited access to table column statistics detected. Please setup" +
```

Comment:
We should probably use `PrintVerbose` here--the cases where we `PrintInfo` for missing helpers seem to be in the (unused) collector reports. If this is running regularly it would be pretty noisy. Plus, superuser is not available on some systems, so there's not much you can do about the warning.

Reply:
Oh, sorry--I missed that this only happens for a test run. It should stay as is.